src/Sulu/Bundle/ResourceBundle/Resources/public/js/components/condition-selection/main.js

```diff
@@ -525,7 +543,9 @@ define([], function() {
                 case 'datetime':
                     return DATETIME_TYPE;
                 default:
-                    this.sandbox.logger.error('Unsupported type "' + type + '" found!');
+                    if (showError !== false) {
```

Comment:
i dont like this ... could we do that in another way? without additional parameter? maybe with a supported types array?=